### DISCUSSION TITLE
Creating a perfect full front lora model headshot

### DISCUSSION THREAD

**Discussion by pes1972** [lllyasviel/stable-diffusion-webui-forge#756] [Q&A]
If i want a perfect full front headshot of face only from a lora model which is the best option? Using the model only and different prompts almost never gives a full front face headshot no matter what prompts used. Using controlnet with depth option works fine but often has issues not getting the correct headshape to match the model face and resemblance is not always that good. Would love suggestions :)

**Comment by abzaloff** [comment]
Use Controlnet with the Instant-ID option. There are many videos online on how to do this

**Comment by pes1972** [comment]
> Use Controlnet with the Instant-ID option. There are many videos online on how to do this

Not an option. The Loras i want to use are 1.5 and instant-ID only works with sdxl models. Any other suggestions?
